test/e2e/arktos/kubectl/config-kubectl-e2e-test.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
there are a few scripts in this folder but only one is supposed to be called from command. Consider move the other scripts into a sub-folder.

Reply:
That is a good suggestion. I move the helper scripts to the 'helper' sub-folder. The doc was updated accordingly.